site/cds_rdm/templates/semantic-ui/invenio_notifications/email/community-submission.submit.jinja

```diff
@@ -0,0 +1,58 @@
+{% set submission_request = notification.context.request %}
+{% set community = submission_request.receiver %}
+{% set creator = submission_request.created_by %}
+{% set record = submission_request.topic %}
+{% set request_id = submission_request.id %}
+{% set recipient_full_name = recipient.data.profile.full_name %}
+{% set help_url = "https://cern.service-now.com/service-portal?id=service_element&name=CDS-Service" %}
+{% set community_title = community.metadata.title %}
+{% set record_title = record.metadata.title %}
+{% set creator_name = creator.username %}
+
+{# TODO: use request.links.self_html when issue issue is resolved: https://github.com/inveniosoftware/invenio-rdm-records/issues/1327 #}
+{% set request_link = "{ui}/me/requests/{id}".format(
+    ui=config.SITE_UI_URL, id=request_id
+    )
+%}
+
+{%- block subject -%}
+    [CDS] New Submission: "{{ record_title }}" in {{ community_title }} community
```

Comment:
Very minor: will the extra space be added to the subject? I would remove it just to be sure.
Maybe lowercase `submission`:
```suggestion
[CDS] New submission: "{{ record_title }}" in {{ community_title }} community
```